context/app/static/js/stores/useSavedEntitiesStore.js

```diff
@@ -16,6 +16,37 @@ const useSavedEntitiesStore = create(
         set((state) => {
           delete state.savedEntities[entityUuid];
         }),
+      savedLists: {},
+      createList: ({ title, description }) =>
+        set((state) => {
+          state.savedLists[title] = {
+            // the Donor, Sample, and Datasets are objects to avoid duplicates. Normally they would be sets, but objects work better with local storage
+            Donor: {},
+            Sample: {},
+            Dataset: {},
```

Comment:
Prefer is just keep keys lowercase, if possible.

Reply:
As we discussed, keeping this for now, but we may revisit later.